packages/core/src/components/Pagination/Select.tsx

```diff
@@ -1,27 +1,21 @@
 import { useState } from "react";
-import { clsx } from "clsx";
-import {
-  HvBaseDropdown,
-  HvSelectionList,
-  HvListItem,
-  HvPanel,
-  HvTypography,
-} from "../index";
-import { StyledRoot } from "./Select.styles";
+import { HvBaseDropdown, HvSelectionList, HvListItem, HvPanel } from "..";
+import { useClasses } from "./Select.styles";
 
 export const Option = ({ children, ...others }) => (
   <HvListItem {...others}>{children}</HvListItem>
 );
 
 const HvSelect = ({
```

Comment:
Noticed the following in the DS3, the extension seems to be bigger:

<img width="542" alt="Screenshot 2023-06-16 at 12 05 04" src="https://github.com/lumada-design/hv-uikit-react/assets/43220251/6d340f87-36a3-4a19-a01b-4554f05432dc">

Reply:
I can't reproduce this, in any browser 🤔

Reply:
Sorry, it depends on the number selected and it was already happening in master but I didn't notice the first time around 😅 My bad